.github/workflows/release-branch.yml

```diff
@@ -44,6 +44,11 @@ jobs:
         echo "yarnPath: '$TMPBIN/yarn.js'" >> .yarnrc.yml
         git update-index --skip-worktree -- .yarnrc.yml
 
+    - name: 'Store the secrets'
+      run: |
+        printf "${{secrets.SIGN_PRIVATE_KEY}}" > /tmp/yarn.key
+        printf "${{secrets.SIGN_PUBLIC_KEY}}" > /tmp/yarn.pem
```

Comment:
I wouldn't store the public key as secret, it's a better idea to derive it from the private key – or store it in clear.

Reply:
I kept it as a secret to make clear both values were linked and needed to be updated should we need to change it (rather than hardcode it in a separate place), but true, it could be derived within the script itself 🤔